src/components/App.tsx

```diff
@@ -2130,8 +2145,9 @@ class App extends React.Component<AppProps, AppState> {
     }
 
     this.setState({
-      selectedElementIds: Object.fromEntries(
-        textElements.map((el) => [el.id, true]),
+      selectedElementIds: makeNextSelectedElementIds(
+        Object.fromEntries(textElements.map((el) => [el.id, true])),
```

Comment:
Single reduce is more performant than first creating array and then transforming it into object

Reply:
this is not a hot path so it's fine